automation/vars/hci-adoption.yaml

```diff
@@ -14,12 +14,12 @@ vas:
             src_file: values.yaml
         build_output: nncp.yaml
 
-      - path: examples/va/hci/control-plane
+      - path: examples/va/hci/control-plane/networking
         wait_conditions:
           - >-
-            oc -n openstack wait nncp
-            -l osp/nncm-config-type=standard
-            --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured
+            oc -n metallb-system wait pod
+            -l app=metallb -l component=speaker
+            --for condition=Ready
```

Comment:
This doesn't really check whether the resources created in this stage are in fact ready.  Unfortunately, those resources (`NetConfig`, `Net-Attach-Def`, `L2Advertisement` and `IPAddressPool`) do not have a status that can be examined.  But our CI validation requires that we have at least one `wait_conditions` listed here, so I've opted to use MetalLB speaker pods as the target here...but it's really just a placeholder.